js/src/components/picc-meter.js

```diff
@@ -0,0 +1,193 @@
+var CLASS_PREFIX = 'picc-meter-';
+
+window.PICCMeter = document.registerElement('picc-meter', {
```

Comment:
I'm interested in why these would be globals if you're using browserify. I'd expect that you would require this file in every file that depended on it and export a local variable here.